packages/pf-codemods/README.md

```diff
@@ -294,6 +294,10 @@ Out:
 />
 ```
 
+### react-dropzone-warn-upgrade [(#7926)](// https://github.com/patternfly/patternfly-react/pull/7926)
+
+The `react-dropzone` dependency used with FileUpload, MultipleFileUpload, and CodeEditor has been updated from version 9 to version 14.
```

Comment:
What might a consumer need to do with this change?

Reply:
From the PR itself, other than updating the version it looks like there were some type updates. Other than possibly mentioning those in the warning message, we could just state that the version bump may require additional breaking changes to be made (though it's a bit generic)

Reply:
Though part of that may be resolved with https://github.com/patternfly/pf-codemods/issues/142

Reply:
Hm, I wonder if this and a PR for #142 are both needed 🤔

Reply:
right... i wonder if the language from this can be merged in (if we still want to talk about version numbers)

Reply:
Yeah that would make sense to me. Want me to do that under a followup?